MM 2.16.0 / 2.17.0 - Newsfeed not working more

I get this error today, but i dont now why i get it, someone there can help. ???
the url work fine.
```
[01.10.2021 13:33.29.814] [ERROR] Newsfeed Error. Could not fetch newsfeed:  https://www.bt.dk/bt/seneste/rss FetchError: request to https://www.bt.dk/bt/seneste/rss failed, reason: certificate has expired
    at ClientRequest.<anonymous> (/home/pi/MagicMirror/node_modules/node-fetch/lib/index.js:1461:11)
    at ClientRequest.emit (events.js:315:20)
    at TLSSocket.socketErrorListener (_http_client.js:426:9)
    at TLSSocket.emit (events.js:315:20)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  type: 'system',
  errno: 'CERT_HAS_EXPIRED',
  code: 'CERT_HAS_EXPIRED'
}
```

The config.js code

```
    {
      module: 'newsfeed',
      position: 'bottom_bar',
      config: {
        feeds: [
          {
            title: 'BT',
            url: 'https://www.bt.dk/bt/seneste/rss'
          }
        ],
        showSourceTitle: true,
        showPublishDate: true,
        broadcastNewsFeeds: true,
        broadcastNewsUpdates: true
      }
    },
```

Same issues here with 2.16.0 in the newsfeed module and the calendar module. Checked the certificates and they are still valid. Interestingly just specific certificates are marked as `expired`.

`
[01.10.2021 14:47.49.143] [ERROR] Newsfeed Error. Could not fetch newsfeed:  https://stadt-bremerhaven.de/feed FetchError: request to https://stadt-bremerhaven.de/feed failed, reason: certificate has expired
0|mm       |     at ClientRequest.<anonymous> (/home/pi/MagicMirror/node_modules/node-fetch/lib/index.js:1461:11)
0|mm       |     at ClientRequest.emit (events.js:315:20)
0|mm       |     at TLSSocket.socketErrorListener (_http_client.js:426:9)
0|mm       |     at TLSSocket.emit (events.js:315:20)
0|mm       |     at emitErrorNT (internal/streams/destroy.js:92:8)
0|mm       |     at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
0|mm       |     at processTicksAndRejections (internal/process/task_queues.js:84:21) {
0|mm       |   type: 'system',
0|mm       |   errno: 'CERT_HAS_EXPIRED',
0|mm       |   code: 'CERT_HAS_EXPIRED'
0|mm       | }
`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,12 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ❤️ **Donate:** Enjoying MagicMirror²? [Please consider a donation!](https://magicmirror.builders/donate) With your help we can continue to improve the MagicMirror²
 
+## [2.17.1] - 2021-10-01
+
+### Fixed
+
+- Fixed error when accessing letsencrypt certificates
+
 ## [2.17.0] - 2021-10-01
 
 Special thanks to the following contributors: @apiontek, @eouia, @jupadin, @khassel and @rejas.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "magicmirror",
-	"version": "2.17.0",
+	"version": "2.17.1",
 	"description": "The open source modular smart mirror platform.",
 	"main": "js/electron.js",
 	"scripts": {

--- a/package.json
+++ b/package.json
@@ -67,7 +67,7 @@
 		"suncalc": "^1.8.0"
 	},
 	"optionalDependencies": {
-		"electron": "^13.5.0"
+		"electron": "^13.5.1"
 	},
 	"dependencies": {
 		"colors": "^1.4.0",

